check-branch-name/action.yml

```diff
@@ -0,0 +1,22 @@
+name: "Check if the branch name follows our naming convention."
+description: "Action that checks if the name of the branch is following the our branch naming convention."
+
+runs:
+  using: composite
+  steps:
+    - uses: jungwinter/split@master
+      id: split
+      with:
+        msg: ${{ env.BRANCH_NAME }}
+        separator: "/"
+        maxsplit: -1
+
+    - name: Check branch name has slash
+      shell: bash
+      if: contains('/', env.BRANCH_NAME) == false
+      run: echo "[ERROR] Branch name ${{ env.BRANCH_NAME }} does not contain a prefix and a backslash." && exit 1
+
+    - name: Check branch prefix naming
+      shell: bash
+      if: contains(fromJson('["fix", "feat", "junk", "maint", "doc", "no-ci", "testing", "release"]'), steps.split.outputs._0) == false
+      run: echo "[ERROR] Branch name ${{ env.BRANCH_NAME }} prefix is not valid." && exit 1
```

Comment:
I would suggest to use the recently added `pyansys/actions/logging` action. It allows to send ANSI color logging messages that can be quickly identified through  all the actions report.